knowledge-base/old-webforms-viewer-messes-rendering-on-mobile.md

```diff
@@ -0,0 +1,41 @@
+---
+title: The Old ASP.NET Webforms ReportViewer Messes Up Reports Rendering on iPad and Other Mobile Devices
+page_title: The Old ASP.NET Webforms ReportViewer Messes Up Reports Rendering on iPad and Other Mobile Devices
+description: "Learn how to handle the situation when the old ASP.NET Webforms ReportViewer messes up the rendering of the reports on iPad and other mobile devices."
+slug: old-webforms-viewer-messes-rendering-on-mobile
+tags: telerik, reporting, troubleshoot, asp, net, webforms, old, reportviewer, messes, up, rendering, reports, on, ipad, mobile
+published: True
+type: troubleshooting
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Version</td>
+			<td>Q2 2012 and newer</td>
+		</tr>
+	        <tr>
+			<td>Report Viewers</td>
+			<td>WinForms, WPF, ASP.NET</td>
```

Comment:
Related only to the legacy AAP.NET WebForms Viewer

Include a note about this so that it is clear to the readers. Most of them don't use this viewer anymore and we recommend them to migrate to the Html5 Viewer